src/jdk.jstatd/share/classes/sun/tools/jstatd/Jstatd.java

```diff
@@ -80,6 +80,8 @@ public static void main(String[] args) {
         int rmiPort = 0;
         int argc = 0;
 
+        System.err.println("Warning: jstatd is deprecated for removal in a future release.");
+
```

Comment:
I think we've put "WARNING" in caps in other usages.

Reply:
I also think a period at the end is not necessary.

Reply:
For the Security Manager, the warning was worded a little differently:

"WARNING: The Security Manager is deprecated and will be removed in a future release"

I think that wording is more clear. The current wording could be confusing because JDK 24 is when jstatd is deprecated for removal, and not a future release. The wording above is more definitive for those that may not understand what "deprecated for removal" means. Thus, I suggest:

"WARNING: jstatd is deprecated and will be removed in a future release"

Reply:
@kevinjwalls Are you planning to move to "WARNING" as has been suggested in the previous comments.

Reply:
No, I was just pausing while the pre-submit checks ran again.  I did investigate, the JDK seems fairly evenly split on WARNING vs Warning.  This message should not be around for many releases.